src/storybook/utils/loaders/index.ts

```diff
@@ -0,0 +1 @@
+export * from './loaders';
```

Comment:
Since we're pulling this out anyway, can we split each loader into a separate file?

Reply:
Good idea. I didn't actually define any loaders but instead some "loader creator" functions so I renamed the loaders file. Many more loaders will eventually come so it makes sense to separate them.